src/commands/transaction/update.ts

```diff
@@ -79,20 +79,24 @@ const displayUpdateResult = (response: SuccessResponse) => {
   console.log(chalk.cyan(chalk.bold(`Transaction ID ${oldTransaction.id}`)));
 
   const table = display.table.createCompact();
-  pushUpdateResultRow(
-    "Title:",
-    table,
-    oldTransaction.title,
-    newTransaction.title
-  );
-  pushUpdateResultRow(
-    "Change in amount:",
-    table,
-    oldTransaction.amountChange,
-    newTransaction.amountChange
-  );
+  if (oldTransaction.title !== newTransaction.title) {
+    pushUpdateResultRow(
+      "Title:",
+      table,
+      oldTransaction.title,
+      newTransaction.title
+    );
+  } else if (oldTransaction.amountChange !== newTransaction.amountChange) {
+    pushUpdateResultRow(
+      "Change in amount:",
+      table,
+      oldTransaction.amountChange,
+      newTransaction.amountChange
+    );
+  };
+  
   if (!table.length) {
-    console.log(chalk.bold("No changes applied."));
+    console.log(chalk.bold("Warning: The values you are trying to update is the same as before, no changes applied."));
```

Comment:
Hi Evence,

Your pull request triggered me to think, and I realized that the `pushUpdateResultRow` might not have been named the best way possible. It won't actually push a row if the old value is not the same as the new value. I didn't care much because this function was meant to be one-off and private, so no one else will be using it anyways.

By doing these `if-else` checks outside of the function, we are essentially checking for the same thing twice. My thinking is that maybe we can refactor the function to take in the `oldTransaction` and `newTransaction` objects together with keys to compare, then push to the update table + displaying warnings.

Does that sound better in your opinion? I can take ownership for this refactor, then you can write in the warning message. How does that sound?

Reply:
That sounds good. I will leave it up to you then if you want even though I think it shouldn't take too long. Merge if you can then take over. Thank you!!